packages/typeorm-legacy/src/utils.ts

```diff
@@ -74,3 +74,34 @@ export function parseConnectionConfig(
     return { url: configOrString } as any
   }
 }
+
+function entitiesChanged(
+  prevEntities: any[] | undefined,
+  newEntities: any[]
+): boolean {
+  if (prevEntities?.length !== newEntities?.length) return true
+
+  for (let i = 0; i < prevEntities?.length; i++) {
+    if (prevEntities[i] !== newEntities[i]) return true
+  }
+
+  return false
+}
+
+export async function updateConnectionEntities(
+  connection: Connection,
+  entities: any[]
+) {
+  if (!entitiesChanged(connection.options.entities, entities)) return
+
+  // @ts-expect-error
+  connection.options.entities = entities
+
+  // @ts-expect-error
+  connection.buildMetadatas()
+
+  if (connection.options.synchronize !== false) {
+    console.warn("[next-auth] ADAPTER_TYPEORM_UPDATING_ENTITIES")
```

Comment:
```suggestion
    console.warn(
      "[next-auth][warn][adapter_typeorm_updating_entities]",
      "\nhttps://next-auth.js.org/warnings#adapter_typeorm_updating_entities"
    )
```
Just a small nitpick, to make this easier for people, let's add a link!

Reply:
I had just thought of that and was going to add it when I had the time. Great 👍